raft.go

```diff
@@ -757,7 +757,10 @@ func (r *raft) reset(term uint64) {
 }
 
 func (r *raft) appendEntry(es ...pb.Entry) (accepted bool) {
-	li := r.raftLog.lastIndex()
+	li, lt := r.raftLog.tip()
+	if r.Term < lt {
```

Comment:
after the first successful `appendEntry` `lt == r.Term`, because `es[i].Term = r.Term` on line 765. So, I don't think this check should be here. It probably should happen at the end of `raft.newRaft()`.

Reply:
+1